src/eos/eosPlugin.ts

```diff
@@ -79,10 +81,16 @@ export class EosTools implements EdgeCurrencyTools {
 
     const currencyInfoType = this.currencyInfo.walletType.replace('wallet:', '')
     if (type === currencyInfoType) {
-      let entropy = Buffer.from(this.io.random(32)).toString('hex')
-      const eosOwnerKey = PrivateKey.from(entropy).toWif()
-      entropy = Buffer.from(this.io.random(32)).toString('hex')
-      const eosKey = PrivateKey.from(entropy).toWif()
+      const ownerKeyEntropy = Buffer.from(this.io.random(32)).toString('hex')
```

Comment:
The bytes / buffer / hex conversion is a bit weird here. Can their `Bytes` class just take a Uint8Array, as returned by the `random` function? If we really need to go hex, use `base16` from rfc4648. Maybe also put the whole random -> Bytes thing on one line, so we aren't converting halfway on one line, and then converting more on the next.